DebugUtils/GTMSynchronizationAsserts.m

```diff
@@ -44,7 +44,9 @@ - (instancetype)initWithSynchronizationObject:(id)object
 
     if (!allowRecursive) {
       BOOL isTopLevelSyncScope = (numberOfSyncingFunctions == 0);
```

Comment:
Let's use: `BOOL __unused isTopLevelSyncScope = ...`

for all these instead.

Reply:
C'est compliquer on dirait